mautic/mautic
# SMS Channel

## Description
This PR adds a new leads communication channel to our Campaign Builder. You can now choose "Send SMS Message" as an action in the builder, and when a user gets to that point, Mautic will send them an automated text message using the awesome Twilio SMS service.

Note: This PR includes the Universal DNC PR (#1513), as it's a requirement for getting this merged.

## Steps to test

- Create a Twilio test account at http://twilio.com. Alternatively, if you're on the Mautic team, you can message me on Slack and get the test account credentials that we've set up already.
- Enter your Account SID as username and Auth Token as password, and the twilio account phone number in your Mautic Configuration.
- Create a new list or form campaign, and add the "Send SMS Message" action to it.
- Submit a lead to your campaign (either by list or submitting a form) using a US-based number (sorry, no international at this point), then build the campaign and trigger it to make it send the text. You should receive the SMS message that you entered in the Campaign Builder.
- Reply with the text "STOP" to the message you received, and you'll receive another message that tells you you've been unsubscribed.
